Simplenote/src/androidTest/java/com/automattic/simplenote/integration/TagsActivityTest.kt

```diff
@@ -30,6 +31,15 @@ import org.junit.runner.RunWith
 class TagsActivityTest : BaseUITest() {
```

Comment:
Praise (❤️): I love the 2 new empty tags related tests that you added based on the bug that you found, kudos!